utils/hostapi.d.ts

```diff
@@ -86,20 +86,17 @@ export interface AzureHostExtensionApi {
     //#endregion
 }
 
+export interface AppResourceFilter {
+    type: string;
+    kind?: string;
+    tags?: Record<string, string>;

```

Comment:
ARM tags I assume? Does a resource need to match one or all of them? (Should probably add a comment to clarify that too)

Reply:
ARM tags. It needs to match all of them, will add comments.